config/locales/server.en.yml

```diff
@@ -2236,7 +2236,7 @@ en:
 
     min_title_similar_length: "The minimum length of a title before it will be checked for similar topics."
 
-    desktop_category_page_style: "This setting determines the visual layout of the /categories page on desktop. It includes options such as displaying subcategories with featured topics, showing the latest topics, or presenting top topics. The chosen style will influence how users interact and navigate through categories on the site."
+    desktop_category_page_style: "This setting determines the visual layout of the /categories page. It includes options such as displaying subcategories with featured topics, showing the latest topics, or presenting top topics. The chosen style will influence how users interact and navigate through categories on the site."
```

Comment:
Might be worth renaming the setting since it no longer applies to `desktop` anymore?

Reply:
I considered that but didn't do it because we're referencing this site setting in various official and customer plugins as seen in [this search result](https://github.com/search?q=org%3Adiscourse%20desktop_category_page_style&type=code).